.eslintrc.base.json

```diff
@@ -3,7 +3,6 @@
     "browser": true
```

Comment:
It looks like with these changes, there are several rules that are turned off that don't affect anything anymore. I was able to delete these rules and the lint check still passed:
```diff
diff --git a/.eslintrc.base.json b/.eslintrc.base.json
--- a/.eslintrc.base.json
+++ b/.eslintrc.base.json
@@ -19,33 +19,21 @@
   },
   "plugins": ["@typescript-eslint", "eslint-plugin-tsdoc"],
   "rules": {
-    "@typescript-eslint/await-thenable": "off",
     "@typescript-eslint/ban-ts-ignore": "off",
     "@typescript-eslint/ban-ts-comment": "off",
     "@typescript-eslint/ban-types": "off",
     "@typescript-eslint/camelcase": "off",
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/explicit-module-boundary-types": "off",
-    "@typescript-eslint/explicit-member-accessibility": "off",
     "@typescript-eslint/no-empty-function": "off",
     "@typescript-eslint/no-unused-vars": "off",
-    "@typescript-eslint/no-misused-promises": "off",
     "@typescript-eslint/no-use-before-define": "off",
     "@typescript-eslint/no-var-requires": "off",
-    "@typescript-eslint/prefer-regexp-exec": "off",
-    "@typescript-eslint/require-await": "off",
-    "@typescript-eslint/unbound-method": "off",
-    "class-methods-use-this": "off",
     "@typescript-eslint/no-redeclare": "off",
-    "global-require": "off",
     "curly": "error",
     "no-global-assign": "warn",
     "react/no-danger": "warn",
-    "import/no-cycle": "off",
     "import/no-anonymous-default-export": "off",
-    "import/prefer-default-export": "off",
-    "jsx-a11y/mouse-events-have-key-events": "off",
-    "max-classes-per-file": "off",
     "monorepo/no-internal-import": "off",
     "no-console": [
       "warn",
@@ -54,24 +42,13 @@
       }
     ],
     "no-debugger": "warn",
-    "no-param-reassign": "off",
-    "no-plusplus": "off",
-    "no-restricted-syntax": "off",
     "no-undef": "error",
     "no-underscore-dangle": "warn",
-    "no-use-before-define": "off",
-    "prettier/prettier": "off",
     "react/destructuring-assignment": "error",
-    "react/forbid-prop-types": "off",
-    "react/jsx-filename-extension": "off",
-    "react/jsx-no-bind": "off",
-    "react/jsx-props-no-spreading": "off",
-    "react/no-array-index-key": "off",
     "react/no-unused-prop-types": "error",
     "react/no-unused-state": "error",
     "react/prefer-stateless-function": "error",
     "react/prop-types": "error",
-    "react/require-default-props": "off",
     "spaced-comment": [
       "error",
       "always",
```